islandora_checksum.module

```diff
@@ -214,17 +300,34 @@ function islandora_checksum_batch_finished($success, $results, $operations) {
  *   Array of DSIDs, if not empty then only these DSIDs will get checksums.
  */
 function islandora_checksum_unpack_dsid_filter() {
-  if (strlen(variable_get('islandora_checksum_dsids_to_check', ''))) {
-    $datastreams_to_check = explode(',',
-      variable_get('islandora_checksum_dsids_to_check', ''));
-  }
-  else {
-    $datastreams_to_check = array();
-  }
-  // Make sure there are no spaces around the DSIDs, so we can get a
-  // match in in_array().
-  foreach ($datastreams_to_check as &$dsid) {
-    $dsid = trim($dsid);
+  $datastreams_to_check = &drupal_static(__FUNCTION__);
+  if (!$datastreams_to_check) {
```

Comment:
As [an empty array is a potential value here](https://github.com/Islandora/islandora_checksum/pull/39/files#diff-a379cac1a58ce40fb083aa204ef795daR310), we could end up getting in here a fair bit... May be better to do an explicit `NULL` check (adding the default to the `drupal_static()` call, if necessary)?